### PR TITLE
[i18n] - Add react-i18n package support

### PR DESCRIPTION
**Pull request by pietro-maximoff**
https://sovryn.atlassian.net/browse/SOV-787

### CODE REVIEW

**Review by soulBit — APPROVED** [review]

File: apps/frontend/package.json [hunk 1]
```diff
@@ -13,8 +13,11 @@
     "@sovryn/ui": "*",
     "classnames": "2.3.2",
     "ethers": "5.7.1",
+    "i18next": "22.0.4",
+    "i18next-browser-languagedetector": "^7.0.0",
```
Comment (creed-victor) [inline apps/frontend/package.json]:
```suggestion
    "i18next-browser-languagedetector": "7.0.0",
```

File: apps/frontend/src/app/5_pages/App/App.tsx [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
   const [isOpen, toggle] = useReducer(p => !p, false);
   const { connectWallet, disconnectWallet, wallets, pending } =
     useWalletConnect();
 
+  const [currentLang, setCurrentLang] = useState(i18next.language);
+
+  const changeLanguage = lng => {
+    i18next.changeLanguage(lng);
+    setCurrentLang(lng);
+  };
```
Comment (creed-victor) [inline apps/frontend/src/app/5_pages/App/App.tsx]:
```suggestion
  const changeLanguage = useCallback((lng: string) => () => {
    i18next.changeLanguage(lng);
    setCurrentLang(lng);
  }, []);
```

File: apps/frontend/src/app/5_pages/App/App.tsx [hunk 3]
```diff
@@ -31,18 +42,15 @@ function App() {
...
-            <div className="my-2" onClick={noop}>
-              Dropdown Item 3
-            </div>
-          </div>
+        <Dropdown text={currentLang.toUpperCase()} className="my-4">
+          <Menu>
+            {languages.map(lng => (
+              <MenuItem
+                text={lng.toUpperCase()}
+                onClick={() => changeLanguage(lng)}
```
Comment (creed-victor) [inline apps/frontend/src/app/5_pages/App/App.tsx]:
```suggestion
                onClick={changeLanguage(lng)}
```

**Review by creed-victor — CHANGES_REQUESTED** [review]

File: apps/frontend/src/app/5_pages/App/App.tsx (hunk 3 above)
Comment (pietro-maximoff) [reply apps/frontend/src/app/5_pages/App/App.tsx]: will get the error `Type 'void' is not assignable to type 'MouseEventHandler<Element> | undefined'.ts` if I change this

Comment (creed-victor) [reply apps/frontend/src/app/5_pages/App/App.tsx]: Even when `changeLanguage` is changed to?:  
```
const changeLanguage = useCallback((lng: string) => () => {
    i18next.changeLanguage(lng);
    setCurrentLang(lng);
  }, []);
```

**Comment by soulBit** [comment]
validated review comments from Victor and fixed last unresolved one, merging since he is unavailable for re-review

**Pull request merged**